Compiler/runtime/systemimpl.c

```diff
@@ -2903,7 +2903,7 @@ int SystemImpl__covertTextFileToCLiteral(const char *textFile, const char *outFi
   errno = 0;
   fout = fopen(outFile, "w");
   if (!fout) {
-    printf("SystemImpl__covertTextFileToCLiteral could not create file. The total file name might be too long. Error %d \n", errno);
+    fprintf(stderr , "SystemImpl__covertTextFileToCLiteral failed: %s Maybe the total file name is too long.", strerror(errno));
```

Comment:
This could still use the error module. If you print to stdout or stderr, the user cannot see it when scripting OMC